### DISCUSSION TITLE
Data Source issue

### DISCUSSION THREAD

**Discussion by DanGMaher** [code4recovery/12-step-meeting-list#1496] [Q&A]
I have a meeting on our Intergroup site aaspokane.org that syncs with area92aa.org.  The meeting "The Way Out Group Spokane" is correct on both sites.  It continues to appear as a meeting that needs to sync daily.  I have deleted it from both sites and added it to aaspokane.org without affecting this issue.  Not a big deal because the meeting is correct on both sites, but it would be nice not to get emails

**Comment by kiyote33** [comment]
Hi Dan,
There's probably some slight difference in the way the meetings are stored in their respective database. Probably some hidden character that can't be discerned by the naked eye. You're in luck though as we will be publishing a major maintenance release which we hope will resolve or at least provide more information as to why the meeting is showing up on the change detected email. Hang in there for a few days, and let us know if the upcoming 3.15 release fixes this issue for you.
